internal/xds/translator/testdata/out/xds-ir/basic-auth.listeners.yaml

```diff
@@ -15,11 +15,17 @@
           maxConcurrentStreams: 100
         httpFilters:
         - disabled: true
-          name: envoy.filters.http.basic_auth
+          name: envoy.filters.http.basic_auth/securitypolicy/default/policy-for-http-route-1
           typedConfig:
             '@type': type.googleapis.com/envoy.extensions.filters.http.basic_auth.v3.BasicAuth
             users:
               inlineBytes: dXNlcjE6e1NIQX10RVNzQm1FL3lOWTNsYjZhMEw2dlZRRVpOcXc9CnVzZXIyOntTSEF9RUo5TFBGRFhzTjl5blNtYnh2anA3NUJtbHg4PQo=
+        - disabled: true
```

Comment:
why this happen?
cc @zhaohuabing

Reply:
https://github.com/envoyproxy/gateway/issues/2947#issuecomment-2675920727